examples/cosmos/main.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+
+	"github.com/schwartzmx/gremtune"
+)
+
+var panicOnErrorOnChannel = func(errs chan error) {
```

Comment:
Would rename it to `panicOnChannelError`

Reply:
But it should panic in case there was an error posted on the channel.

Reply:
Ok, I understand.